MissionToPsyche/tungsten_11a_vr_game-se
# Next player menu

This PR adds the UI and scripting for the next player menu, and updates the related scripts to make it function properly. The main changes include:
- New 'NextPlayerMenu' scene
- New 'NextPlayerMenuController.cs' to manage the menu functionality.
- Added 'ResetForNextPlayer' methods in 'VRPlanetMapInteraction.cs' and 'SceneClick.cs' to reset the number of scenes visited for the next player when they click continue.
- Updated the event mode tracking functionality to redirect to this menu rather than the main menu.

### TO TEST
1. Start game at main menu
2. Click Event Mode
3. Enter a scene and time limit (I've been testing with 1 scene and 1 minute)
4. Either wait for the time limit or click to go to other scenes until you reach the scene limit
5. Make sure the next player menu pops up, and the message for the next player is correct. 
6. Continue as next player
7. Repeat steps 4 and 5
